templates/webpack-default/app/views/home/index.xml

```diff
@@ -0,0 +1,14 @@
+<Alloy>
+	<Tab title="Home">
+		<Window title="Home">
+			<View id="wrapper">
+				<Label class="fa-solid search">edit</Label>
+				<Label id="editLabel">
+					<AttributedString class="as">
+						Edit home.js to change this view!
```

Comment:
This should be `views/home/index.xml`. Although technically there are three different files that make up the view, but that probably blows up the example.

Reply:
Doi of course!